src/imap/model.rs

```diff
@@ -69,7 +116,30 @@ impl<'a> ImapConnector<'a> {
         Ok(())
     }
 
-    pub fn add_flags(&mut self, mbox: &str, uid_seq: &str, flags: &str) -> Result<()> {
+    /// Add the given flags to the given mail.
+    ///
+    /// # Example
+    /// ```no_run
+    /// use himalaya::imap::model::ImapConnector;
+    /// use himalaya::config::model::Account;
+    /// use himalaya::flag::model::Flags;
+    /// use imap::types::Flag;
+    /// 
+    /// fn main() {
+    ///     let account = Account::default();
+    ///     let mut imap_conn = ImapConnector::new(&account).unwrap();
+    ///     let flags = Flags::from(vec![Flag::Seen]);
+    ///
+    ///     // Mark the message with the UID 42 in the mailbox "rofl" as "Seen"
+    ///     imap_conn.add_flags("rofl", "42", flags).unwrap();
+    ///
+    ///     imap_conn.logout();
+    /// }
+    /// ```
+    pub fn add_flags(&mut self, mbox: &str, uid_seq: &str, flags: Flags) -> Result<()> {
+
```

Comment:
Why adding a new line here? It does not really fit the global style of the code base.

Reply:
Should be fixed now.